ValueError('Invalid limits given: (((y, 0, 10), (f, 9, 9)),)',)
